### DISCUSSION TITLE
How can I remove specific commands from showing up in *help? And how can I remove achievement pictures from showing up?

### DISCUSSION THREAD

**Discussion by priyam-raj** [Androz2091/AtlantaBot#475] [Q&A] [answered]
Hello Androz, I have two questions:

1. How can I remove specific commands from showing up in the *help embed because I didn't find anything in commands/help.js (screenshot 1) e.g. I want to remove *fortniteshop from showing up there for now as my API is not yet approved. 
2. How can I remove achievement pictures from showing up every time someone achieves something? I would like to change it to something else later and so nothing for now (screenshot 2) 

<img width="904" alt="Screenshot 1" src="https://user-images.githubusercontent.com/26673536/118393380-78828c00-b65c-11eb-835c-a06053b3c46d.png">

<img width="537" alt="Screenshot 2" src="https://user-images.githubusercontent.com/26673536/118393442-cd260700-b65c-11eb-9fdd-723e82c34476.png">

**Comment by Androz2091** [comment, marked as answer]
Hello,

1) Turn the `enabled` property of the command to `false`. https://github.com/Androz2091/AtlantaBot/blob/master/commands/General/fortnite.js#L10

2) Edit the `progress` so they all match `total` in the `base/User.js` file (for example, here https://github.com/Androz2091/AtlantaBot/blob/master/base/User.js#L28, change the value to 1). Then clear your database and restart the bot. All the achievements will be considered are claimed for everyone once their accounts are created and the messages will no longer appear.
